plugins/destination/postgresql/client/client_test.go

```diff
@@ -6,38 +6,43 @@ import (
 	"testing"
 	"time"
 
+	"github.com/cloudquery/plugin-sdk/plugins"
 	"github.com/cloudquery/plugin-sdk/schema"
 	"github.com/cloudquery/plugin-sdk/specs"
-	"github.com/google/uuid"
 	"github.com/rs/zerolog"
 )
 
-func getTestData() map[string]interface{} {
+func getTestData() *schema.DestinationResource {
 	// because data is sent over the wire encoded in json we need to use strings, numbers, objects, arrays, booleans and nulls
 	// to test everything correctly
-	return map[string]interface{}{
-		"_cq_id":              uuid.New().String(),
-		"_cq_parent_id":       nil,
-		"_cq_source_name":     "test_source",
-		"_cq_sync_time":       "2022-09-02T20:57:55.449238",
-		"id":                  uuid.New().String(),
-		"bool_column":         true,
-		"int_column":          float64(3),
-		"float_column":        float64(3),
-		"uuid_column":         "9a6011b7-c5ee-4b55-95a6-37ce5e02a5a0",
-		"string_column":       "test",
-		"string_array_column": []interface{}{"test", "test2"},
-		"int_array_column":    []interface{}{float64(1), float64(2), float64(3)},
-		"timestamp_column":    "2019-01-01T00:00:00",
-		"interval_column":     "01:02:03",
-		"json_column":         map[string]interface{}{"1": float64(1), "test": "test"},
-		"uuid_array_column":   []interface{}{"1a6011b7-c5ee-4b55-95a6-37ce5e02a5a0", "9a6011b7-c5ee-4b55-95a6-37ce5e02a5a0"},
-		"inet_column":         "1.1.1.1",
-		"inet_array_column":   []interface{}{"8.8.8.8/0"},
-		"cidr_column":         "0.0.0.0/24",
-		"cidr_array_column":   []interface{}{"0.0.0.0/24", "0.0.0.0/16"},
-		"mac_addr_column":     "00:00:00:00:00:ab",
-	}
+	return nil
```

Comment:
Looks like this still needs to be implemented

Reply:
Moved it to SDK part